### PR TITLE
Feature/link to api

### PR DESCRIPTION
**Pull request by WouterKvG**
@bartist @steffansluis Please check if it works on your machine. (In combination with https://github.com/feedbackfruits/bepstore-ui-goals/pull/4)

### CODE REVIEW

**Comment by steffansluis** [comment]
The code looks good but I can't seem to get `/mygoals` to work. The problem seems to be that the list of goals is empty (even though it isn't in the endpoint on api), so it probably messes up somewhere in the serialization/deserialization.

**Comment by steffansluis** [comment]
@WouterKvG Added the moment-transform to bower.json seems to have broken dependency resolution. I tried to fix it, but it doesn't seem to be working. Any chance you could take a look?

**Comment by WouterKvG** [comment]
@steffansluis I saw a moment field type definition in fbf-ui. Maybe we should use that instead?

**Comment by steffansluis** [comment]
@WouterKvG It seems like if that also solves the problem, it would be the cleaner way to do it.

**Comment by WouterKvG** [comment]
@steffansluis Can I merge?

File: app/routes/goal.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
 import Ember from 'ember';
-import mockMaker from '../utils/mockMaker';
 
 export default Ember.Route.extend({
   model(params) {
-    return mockMaker().oneGoal(params.goal_id);
+    return this.get('store').findRecord('activity', params.goal_id);
```
Comment (steffansluis) [inline app/routes/goal.js]: Does this need to be activity, or can it be goal?

**Comment by steffansluis** [comment]
LGTM

**Pull request merged**